code/datums/storage/subtypes/modsuit.dm

```diff
@@ -0,0 +1,9 @@
+/datum/storage/modsuit
+
+/datum/storage/modsuit/on_attackby(datum/source, obj/item/thing, mob/user, params)
+	var/obj/item/mod/control/mod = parent?.resolve()
+	var/obj/item/mod/core/plasma/plasma_mod_core = mod.core
+	if(istype(plasma_mod_core) && (thing.type in plasma_mod_core.charger_list))
+		return
+
+	return ..()
```

Comment:
i dislike this solution, what prevents the first one from working alone?

Reply:
First part alone should be enough, it won't do anything about the plasma getting inserted into storage at the same time as recharging but if you don't care about that I can just remove this I guess